crates/beerus-rpc/src/lib.rs

```diff
@@ -2,7 +2,8 @@ use eyre::Result;
 use jsonrpsee::server::{ServerBuilder, ServerHandle};
 use std::net::{AddrParseError, SocketAddr};
 use thiserror::Error;
-mod server;
+pub mod server;
```

Comment:
I had to make them public because otherwise I couldn't use them in the utils to initialize the mocked config, any warning?

Reply:
no warning, looks fine